### PR TITLE
New FORTRAN Cheat Sheet

### PR DESCRIPTION
**Pull request by jumpball**
https://duck.co/ia/view/fortran_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@jumpball Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/fortran.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    "id": "fortran_cheat_sheet",
+    "name": "FORTRAN Cheatsheet",
+    "description": "List of the most important FORTRAN commands",
+
+    "metadata": {
+        "sourceName": "FORTRAN 77 tutorial",
+        "sourceUrl" : "http://web.stanford.edu/class/me200c/tutorial_77/"
+    },
+
+    "aliases": [
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/fortran.json]: Get rid of the aliases seeing as its empty.

File: share/goodie/cheat_sheets/json/fortran.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        "Numbers and Variables",
+        "Logical",
+        "I/O",
+        "Other"
+    ],
+
+    "sections": {
+        "Execution": [
+            {
+                "key": "      PROGRAM name",
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/fortran.json]: Why are there spaces? Get rid of any erroneous white-space in fields.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheatsheet_mobile.png)

File: share/goodie/cheat_sheets/json/fortran.json (hunk 1 above)
Comment (jumpball) [inline share/goodie/cheat_sheets/json/fortran.json]: @GuiltyDolphin It is changed now, no more empty aliases.

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheatsheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=FORTRAN cheatsheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/fortran_cheat_sheet_mobile.png)

**Comment by moollaza** [comment]
@jumpball Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@jumpball this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Comment by moollaza** [comment]
@jumpball thanks again for creating the Fortran Cheat Sheet! You’ve done a really good job here and we'd like you to help make DuckDuckGo even better for yourself and others interested in Fortran.

When you're ready to move on, please: 

- Create a new Instant Answer for searches you do related to Fortran

OR

- Test live Fortran Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Creating-or-Improving-Instant-Answers).

Thanks again!

**Pull request merged**